minifi-nar-bundles/minifi-framework-bundle/minifi-framework/minifi-runtime/pom.xml

```diff
@@ -40,6 +40,24 @@ limitations under the License.
             <groupId>org.apache.nifi.minifi</groupId>
             <artifactId>minifi-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>ro.pippo</groupId>
+            <artifactId>pippo-core</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>ro.pippo</groupId>
+            <artifactId>pippo-tomcat</artifactId>
```

Comment:
It was part due to not knowing enough about the differences between Tomcat and Jetty, as well as having a bit of trouble configuring Jetty to do https. If Jetty is much better suited to our needs I will work through the problems I was having initially.